.github/templates/windows_ci_workflow.yml.in

```diff
@@ -70,7 +81,7 @@ jobs:
         shell: powershell
         run: |
           .\.github\scripts\install_windows_sdk.ps1
-      - uses: actions/download-artifact@v2
+      - uses: seemethere/download-artifact-s3@0504774707cbc8603d7dca922e8026eb8bf3b47b
```

Comment:
btw, what's the permission setting for the s3 bucket? Is it public gettable, wondering if we need to run through some security checks?

Reply:
No it's not publicly accessible, I tried adding `public-read` for ACL settings but I still need to fix IAM policies before enabling that